src/main/java/blusunrize/immersiveengineering/client/manual/ManualElementBottling.java

```diff
@@ -0,0 +1,197 @@
+				++recipePage;
+				if(recipePage >= this.recipes.size())
+					recipePage = 0;
+			}));
+		}
+		super.onOpened(gui, x, y+yOff+2, pageButtons);
+	}
+
+	@Override
+	public void render(PoseStack transform, ManualScreen gui, int x, int y, int mouseX, int mouseY)
+	{
+		highlighted = ItemStack.EMPTY;
+
+		if(!recipes.isEmpty()&&recipePage >= 0&&recipePage < this.recipes.size())
+		{
+			PositionedItemStack[] pStacks = recipes.get(recipePage);
+			int maxX = 0;
+			for(PositionedItemStack pStack : pStacks)
+			{
+				if(pStack!=null)
+				{
+					if(pStack.x() > maxX)
+						maxX = pStack.x();
+					GuiComponent.fill(transform, x+pStack.x(), y+pStack.y(), x+pStack.x()+16, y+pStack.y()+16, 0x33666666);
+
+					if(!pStack.getStackAtCurrentTime().isEmpty())
+					{
+						ManualUtils.renderItemStack(transform, pStack.getStackAtCurrentTime(), x+pStack.x(), y+pStack.y(), true, pStack.amount());
+						if(mouseX >= x+pStack.x()&&mouseX < x+pStack.x()+16&&mouseY >= y+pStack.y()&&mouseY < y+pStack.y()+16)
+							highlighted = pStack.getStackAtCurrentTime();
+					}
+				}
+			}
+			ManualUtils.drawTexturedRect(transform, manual.texture, x+46, y+yOff-13, 16, 10, 0/256f, 16/256f, 226/256f, 236/256f);
+		}
+
+		this.renderHighlightedTooltip(transform, gui, mouseX, mouseY);
+	}
+
+	@Override
+	public boolean listForSearch(String searchTag)
+	{
+		for(PositionedItemStack[] recipe : this.recipes)
+			for(PositionedItemStack pStack : recipe)
+				for(ItemStack stack : pStack.displayList())
+					if(ManualUtils.listStack(searchTag, stack))
+						return true;
+		return false;
+	}
+
+	@Override
+	public int getPixelsTaken()
+	{
+		int maxY = 0;
+		for(PositionedItemStack[] recipe : recipes)
+			for(PositionedItemStack pstack : recipe)
+				maxY = Math.max(maxY, pstack.y());
+		return maxY+18;
+	}
+
+	static Map<Integer, String> fractionStrings = new HashMap<>();
+
+	static
+	{
+		// this is not all the existing fraction symbols, but the most commonly used
+		// ToDo: Maybe use org.apache.commons.lang3.math.Fraction here?
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/2, "½");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/4, "¼");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/4*3, "¾");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/9, "⅑");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/3, "⅓");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/3*2, "⅔");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/8, "⅛");
+		fractionStrings.put(FluidAttributes.BUCKET_VOLUME/8*3, "⅜");
+	}
+
+	private String getBucketFraction(int amount)
+	{
+		String ret = "";
+		// if amount is bigger than bucket, consider those as full numbers
+		if(amount > FluidAttributes.BUCKET_VOLUME)
+		{
+			ret += amount/FluidAttributes.BUCKET_VOLUME;
+			amount = amount%FluidAttributes.BUCKET_VOLUME;
+		}
+		// remaining amount
+		if(amount > 0)
+		{
+			// use fraction symbols where possible
+			if(fractionStrings.containsKey(amount))
```

Comment:
I personally find the fractions quite hard to read in-game. What do you think about a client config option to skip this branch and just give a decimal representation for everything?

Reply:
Hrm, I suppose.

I really like their visuals myself, but you have a point.